x-pack/plugins/reporting/index.js

```diff
@@ -139,10 +141,16 @@ export const reporting = (kibana) => {
 
     init: async function (server) {
       const exportTypesRegistry = await exportTypesRegistryFactory(server);
+      const { getClient } = server.plugins.elasticsearch.getCluster('admin');
+      const esClient = createClient(getClient());
```

Comment:
This client (`elasticsearch`) is used now in two spots inside of reporting. I've attached it as a plugin to our namespace, but if that pattern isn't clear let me know and I can alter